Misc/NEWS.d/next/Library/2022-10-09-18-08-08.gh-issue-97971.SaVHTd.rst

```diff
@@ -0,0 +1,3 @@
+Improved :class:`TypeError` message for undefined abstract methods of a
```

Comment:
The NEWS entry must refer to an issue, not a PR. You'll have to create an issue and rename the file to have your issue number in the file name.

Reply:
Thanks, see [4073b1f](https://github.com/python/cpython/pull/97971/commits/4073b1fb4a0afe64c5c5742c75b12d80262bae31).